source/tyr/tyr/resources.py

```diff
@@ -1637,6 +1639,51 @@ def put(self, instance_name):
         return {'action': return_msg}, return_status
 
 
+def check_db():
+    cities_db = sqlalchemy.create_engine(current_app.config['CITIES_DATABASE_URI'])
+    try:
+        cities_db.connect()
+        result = cities_db.execute("SELECT version_num FROM alembic_version")
+        for row in result:
+            return row['version_num']
+    except Exception as e:
+        logging.exception("cities db not created : {}".format(e.message))
+        return None
+
+
+class CitiesGET(flask_restful.Resource):
```

Comment:
Minor: here `CitiesStatus` looks clearer to me.
And below `Cities` as it might handle other methods (we can change it then).